src/components/TVChartContainer/charting_library/charting_library.d.ts

```diff
@@ -0,0 +1,2435 @@
+// Generated by dts-bundle-generator v5.3.0
```

Comment:
this is original file from TV library, right? it can't be moved to `public/charting_library` dir because typescript won't see it? 

just would be good to not move it to our components if possible

Reply:
Removed

Reply:
Probably we will create some utils that uses types from here, or create different version of TvChartContainer (e.g. for synthetics) maybe put it into src/charting_library.d.ts - it's a common way to place typings for external libraries